src/node.h

```diff
@@ -308,7 +308,7 @@ NODE_EXTERN int Start(int argc, char* argv[]);
 
 // Tear down Node.js while it is running (there are active handles
 // in the loop and / or actively executing JavaScript code).
-NODE_EXTERN int Stop(Environment* env);
+NODE_EXTERN int Stop(Environment* env, bool terminate = true);
```

Comment:
Just a few thoughts, nothing blocking:
- Keeping a `NODE_EXTERN int Stop(Environment* env);` overload until the next release instead of using a default argument would make this semver-minor
- It might be more flexible to take a flags argument here than a boolean
- It might be worth adding a note to the doc comment here about what the argument does

Reply:
Refactored it a bit - let me know if this is closer to what you envisioned!

Reply:
Yeah, this looks great! :+1: